bootstrap/Old_BootStrapBackup/shoppingmall_review_seed.py

```
@@ -0,0 +1,298 @@
+]
+PRODUCT_IDS = list(range(1, 43))
+USER_IDS = list(range(1, 6))
+
+
+def generate_review(review_id: int, sentiment: str, product_id: int) -> dict:
+    product_name = (
+        PRODUCT_NAMES[product_id - 1] if product_id <= len(PRODUCT_NAMES) else "상품"
+    )
+
+    if sentiment == "positive":
+        template = random.choice(POSITIVE_TEMPLATES)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)